include/grpcpp/server_impl.h

```diff
@@ -183,6 +184,8 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
       std::shared_ptr<std::vector<std::unique_ptr<grpc::ServerCompletionQueue>>>
           sync_server_cqs,
       int min_pollers, int max_pollers, int sync_cq_timeout_msec,
+      std::vector<std::shared_ptr<::grpc_impl::ExternalConnectionAcceptorImpl>>
```

Comment:
not sure if we are allowed to make this change since this is protected

Reply:
I think it should be fine, as the ServerBuilder reference is handled in this PR and the internal subclass will be fixed once I import this change.

Reply:
Right, it is a breaking change in that sense. What I was trying to say is that I know why it is marked protected and I will fix the internal subclass. I do not think it will affect users as this is not really public API.